index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Bartender Pirate!</title>
+    <meta name="description" content="Arr! How you like yer drink, matey? A pirate bartender app.">
+    <meta name="author" content="Row and Yoli" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+    <link href="https://necolas.github.io/normalize.css/4.1.1/normalize.css" rel="stylesheet" type="text/css">
+    <link href="style.css" rel="stylesheet" type="text/css">
+</head>
+<body>
+	<div class="binding-box">
+		<h2>How ye want yer drink, matey?</h2>
+
+
+	<fieldset id="drink-strong">
+		<h3>Do ye like yer drinks strong?</h3>
+		<input type="radio" name="yes" class="yes">
+		<label for="yes">Yes!</label>
+		<input type="radio" name="no" class="no">
+		<label for="no">Nope!</label>
+	</fieldset>
+
+
+	<fieldset id="drink-salty">
+		<h3>Do ye like it with a salty tang?</h3>
+		<input type="radio" name="yes" class="yes">
+		<label for="yes">Yes!</label>
+		<input type="radio" name="no" class="no">
+		<label for="no">Nope!</label>
+	</fieldset>
+
+
+	<fieldset id="drink-bitter">
+		<h3>Are ye a lubber who likes it bitter?</h3>
+		<input type="radio" name="yes" class="yes">
+		<label for="yes">Yes!</label>
+		<input type="radio" name="no" class="no">
+		<label for="no">Nope!</label>
+	</fieldset>
+
+
+	<fieldset id="drink-sweet">
+		<h3>Would ye like a bit of sweetness with yer poison?</h3>
+		<input type="radio" name="yes" class="yes">
+		<label for="yes">Yes!</label>
+		<input type="radio" name="no" class="no">
+		<label for="no">Nope!</label>
+	</fieldset>
+
+
+	<fieldset id="drink-fruity">
+		<h3>Are ye one for a fruity finish?</h3>
+		<input type="radio" name="yes" class="yes">
+		<label for="yes">Yes!</label>
+		<input type="radio" name="no" class="no">
+		<label for="no">Nope!</label>
+	</fieldset>
+
+	</div>
+    <script src="https://code.jquery.com/jquery-3.1.0.min.js"></script>
+    <script src="app.js"></script>
+</body>
+</html>
```

Comment:
Don't overdue spacing in between after each fieldset. It does add size to your file and is unnecessary.